### ISSUE TITLE
Where is the loss function?

### ISSUE THREAD

**Issue by smilecode-github** [jiaoZ7688/YOLOPX#2]
Hi

I want to train YOLOPX.

But, I can't find the loss function and train.py

When upload your loss function and train.py files?

Thank you.

**Comment by MartinPR307** [comment]
Expecting...

**Comment by lebedyncrs** [comment]
@smilecode-github did you train a model on your data? How did it go in terms on performance?
